salt/states/group.py

```diff
@@ -299,10 +309,20 @@ def absent(name):
     if grp_info:
         # Group already exists. Remove the group.
         if __opts__['test']:
-            ret['result'] = None
-            ret['comment'] = 'Group {0} is set for removal'.format(name)
+            if onlyifempty and grp_info['members']:
+                ret['result'] = True
+                ret['comment'] = ('Group {0} will not be removed since members are '
```

Comment:
Using the future tense here implies that we are definitely going to make changes, we should be using the conditional tense here. Instead of "will not be removed" it should be "would not be removed".